src/components/Facets/Disjunctive/index.js

```diff
@@ -32,7 +32,8 @@ var DisjunctiveFacet = React.createClass( {
                 onClick={ this.toggleSelect.bind( this, facet.name, facetValue.name ) }>
                <input type="checkbox"
                       onClick={ function( e ) { e.preventDefault(); } }
-                      checked={ facetValue.isRefined } />
+                      checked={ facetValue.isRefined }
+                      readOnly/>
```

Comment:
There is a warning generated by React when a component does not implement onChange nor is readonly when doing a controlled component. (a controlled component is a component which value is set with React)